transform-plugins/widgets/ValueMapper-transform.json

```diff
@@ -5,10 +5,16 @@
   "configuration-groups": [
     {
       "properties": [
-        {
-          "widget-type": "textbox",
+	    {
```

Comment:
This is not lined up correctly with the following (closing) brace. It should be moved to the left.